src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -146,93 +170,50 @@ export const updateDonorStatsFromSubmissionCommit = (
   // registration has no buisness here
   if (clinicalType === ClinicalEntitySchemaNames.REGISTRATION) return;
 
+  let updatedDonor = cloneDeep(donor) as Donor;
```

Comment:
typescript isn't inferring that the cloned `donor` is also a `Donor` type?

Reply:
Inference is working, this can be removed